client/src/Components/RegistrationForm/RegistrationForm.tsx

```diff
@@ -0,0 +1,108 @@
+import { Formik, Form, FormikHelpers } from 'formik';
+import { RegistrationFormContainer, RegistrationInput, RegistrationButton, RegistrationFormTitle, RegistrationErrorMessage } from './RegistrationFormStyles';
+import * as Yup from 'yup';
+
+
+interface Values {
+  email: string;
+  password: string;
+  confirmPassword: string;
+}
+const SignupSchema = Yup.object().shape({
+  email: Yup.string()
+    .email("Niewłaściwy adres email")
+    .required("Email jest wymagany!"),
+  password: Yup.string()
+    .min(8, 'Hasło musi mieś conajmniej 8 znaków!')
+    .required('Hasło jest wymagane!'),
+  confirmPassword: Yup.string()
+    .oneOf([Yup.ref('password'), ''], 'Hasło musi się zgadzać!')
+    .required('Powtórzenie hasła jest wyamgane!'),
+});
+
+
+
+export const RegistrationForm = () => {
+  return (
+    <RegistrationFormContainer>
+      <RegistrationFormTitle>Nie masz jeszcze konta?</RegistrationFormTitle>
+      <Formik
+        initialValues={{
+          email: '',
+          password: '',
+          confirmPassword: '',
+        }}
+        validationSchema={SignupSchema}
+        onSubmit={(
+          values: Values,
+          { setSubmitting }: FormikHelpers<Values>
+        ) => {
+          setSubmitting(true);
+
+          fetch('http://localhost:8080/api/users/register', {
+            method: 'post',
+            mode: 'no-cors',
+            headers: {
+              'Accept': 'application/json',
+              'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+              values
+            })
+          }).then((res) => res.json())
+            .then((result) => {
+              alert(result);
+            });
```

Comment:
Przeróbmy to na axiosa i wynieśmy to przed return będzie duzo bardziej czytelnie